optional_requirements.txt

```diff
@@ -8,10 +8,7 @@
 ## of pip from the website, and then possibly install/upgrade setuptools using
```

Comment:
Actually, we are still maintaining two modes of installation, the one based on installing aiida directly with pip, and the old one based on manually installing . 

If one goes for the old installation, then `optional_requirementes.txt` is fully complementary to `requirements.txt`, so we cannot get rid of it, unless we decide not to maintain the old installation mode.

Moreover, so far the two modes have not been fully decoupled. So, after having installed `aiida` in the new way, there is still need to run 
```bash
pip install -r optional_requirements.txt
```
However, the new commits get us much closer to make the last command useless (2-3 more package are still only in `optional_requirement.txt`).